README.md

```diff
@@ -1,26 +1,63 @@
 [![Downloads](https://img.shields.io/packagist/dm/creof/doctrine2-spatial.svg)](https://packagist.org/packages/creof/doctrine2-spatial)
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/creof/doctrine2-spatial)
+[![Documentation Status](https://readthedocs.org/projects/doctrine2-spatial/badge/?version=latest)](https://doctrine2-spatial.readthedocs.io/en/latest/?badge=latest)
 
+Documentation 
+-------------
 
-Doctrine2 multi-platform support for spatial types and functions. Currently MySQL and PostgreSQL with PostGIS are supported. Could potentially add support for other platforms if an interest is expressed.
+The [new documentation](https://doctrine2-spatial.readthedocs.io) explain how to:
 
-Documentation can be found at [here](./doc/index.md)
+* install this doctrine extension,
+* configure this extension,
+* create spatial entities,
+* use spatial functions into your repositories,
+* contribute (and test)
 
-## composer.json
-```javascript
-{
-    "require": {
-    	...
-        "creof/doctrine2-spatial": "~1"
-```
+The documentation contains a glossary of all available types and all available spatial functions.
 
-You will also have to change the version requirement of doctrine to at least 2.3:
-```javascript
+## Project origins
+This useful project was created by Derek J. Lambert. 
+Alexandre Tranchant forked it from [creof/doctrine2-spatial](https://github.com/creof/doctrine2-spatial) 
+because project seems to be non-active since 2017.
 
-        "doctrine/orm": ">=2.3",
-```
+The master release can be used, but be careful of backward incompatibility. 
+
+## Developments in progress
+This fork will upgrade this package to the last doctrine version and the [PHP supported versions](https://www.php.net/supported-versions.php). 
+I would like to release the 2.0.0 version at the end of March.
+
+
+Compatibility
+-------------
+### PHP
+This doctrine extension is compatible with PHP 7.2, 7.3 and 7.4
+Security fixes will follow the [PHP Roadmap](https://www.php.net/supported-versions.php).
+
+### Doctrine dev version
+This extension should be used with the actual doctrine stable version: 2.7
+Continuous integration tests libraries with 2.8.x-dev version. We **ONLY** try to stay compatible with this version, 
+currently. 
+
+### MySQL 5.7 and 8.0
+A lot of functions change their names between this two versions. The [MySQL 5.7 deprecated functions](https://stackoverflow.com/questions/60377271/why-some-spatial-functions-does-not-exists-on-my-mysql-server)
+are not implemented.
+
+### MariaDB 10
+This version is **NOT** compatible with MariaDB version. Some spatial functions seems to work but their results are 
+different from MySQL version (StContains function is a good example). You can contribute, but I suggest to avoid 
+MySql and MariaDb servers, because of [their shortcomings and vulnerabilities](https://sqlpro.developpez.com/tutoriel/dangers-mysql-mariadb/).
```

Comment:
The link to the article should be removed. Because it is a subjective opinion about RDBMS choice (expressed in a rather aggressive tone) and has no reason to be mentioned here. It also happens to be in French which most readers might not be able to read. And finally the way it is introduced make it looks like it is about MariaDB only, but it actually is about both MySQL and MariaDB.

As a matter of fact, the whole "Compatibility" section here is a bit confusing. Topics are mixed between "Compatibility between this lib and a RDBMS" and "Compatibility across RDBMS". The fact that `StContains` returns different result between MySQL and MariaDB does not make this lib incompatible with MariaDB.